compressor/seata-compressor-gzip/src/main/java/io/seata/compressor/gzip/GzipUtil.java

```diff
@@ -28,48 +26,43 @@
  */
 public class GzipUtil {
 
+    private GzipUtil() {
+
+    }
+
     private static final int BUFFER_SIZE = 8192;
 
     public static byte[] compress(byte[] bytes) {
         if (bytes == null) {
             throw new NullPointerException("bytes is null");
         }
-        ByteArrayOutputStream out = null;
-        GZIPOutputStream gzip = null;
-        try {
-            out = new ByteArrayOutputStream();
-            gzip = new GZIPOutputStream(out);
+
+        try (ByteArrayOutputStream out = new ByteArrayOutputStream();
+             GZIPOutputStream gzip = new GZIPOutputStream(out)) {
```

Comment:
The close method of GZIPOutputStream should be called.And the close method of ByteArrayOutputStream actually does nothing.